pkg/topology/kademlia/kademlia.go

```diff
@@ -306,13 +319,13 @@ func (k *Kad) connectNeighbours(wg *sync.WaitGroup, peerConnChan chan<- *peerCon
 
 	sent := 0
 	var currentPo uint8 = 0
-	_ = k.knownPeers.EachBinRev(func(addr swarm.Address, po uint8) (bool, bool, error) {
+
+	_ = k.knownPeers.EachBin(func(addr swarm.Address, po uint8) (bool, bool, error) {
```

Comment:
this changes the iterator to be from deepest to closest, not sure if it is needed (we'd like to connect to shallower peers first, so that depth can gradually go deeper)

Reply:
good catch, this is not needed, a typo.